### PR TITLE
feat(browser): added attach assets to invocation

### LINKED ISSUE #88: Attach assets to invocations
We should let users attach assets to invocations so they can run VerificationTransaction methods in their smart contract (accept payments with invocations).

### PR DESCRIPTION
**Pull request by DalderupMaurice**
## Description
Attach assets to invocations 

Tasks:
- [x] Actual/raw implementation
- [ ] Unit tests
- [ ] Update documentation

## Motivation and Context
Requested feature by several people

## How Has This Been Tested?
Not yet

## Screenshots (if appropriate):
/

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [x] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** guidelines and confirm that my code follows the code style of this project.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Closing issues
Fixes #88

### CODE REVIEW

File: src/renderer/shared/util/getTokens.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  const response = await fetch(TOKENS_URL);
+  const tokens = values(await response.json());
+
+  // eslint-disable-next-line
+  return sortBy(tokens, 'symbol').map((token) => {
+    const tokenDetails = token.networks[networkKey];
+    if (tokenDetails) {
+      return details
+        ? { symbol: token.symbol, ...tokenDetails }
+        : tokenDetails.hash;
```
Comment (mhuggins) [inline src/renderer/shared/util/getTokens.js]: ⚠️ This changes the original behavior of the `getToken` function, and it now results in `undefined` values being a part of the array.

Comment (DalderupMaurice) [reply src/renderer/shared/util/getTokens.js]: Aaah I see 
I used an older branch having that same issue :P 
Took the logic over thinking we had fixed it - not realizing the piece of code was outdated xD 
I originally also used the getTokens but it rendered to be useless for the piece I worked on

**Comment by DalderupMaurice** [comment]
Going to pick this back up tomorrow/later this week =

**Comment by DalderupMaurice** [comment]
Got the next 4 days off so I'll be able to write an SC, deploy & test it 😄

**Comment by DalderupMaurice** [comment]
I've spent many hours testing - but I only get it working once after contract deployment and then it always returns false 😅 
Exact same things works like a charm through neo-python

**Pull request merged**